client/src/components/SyncCard/SyncCard.tsx

```diff
@@ -0,0 +1,74 @@
+import "./SyncCard.css";
+
+import {
+  CheckCircleTwoTone,
+  ClockCircleTwoTone,
+  CloseCircleTwoTone,
+  DeleteOutlined,
+  EditOutlined,
+  EllipsisOutlined,
+} from "@ant-design/icons";
+import { Card, Divider } from "antd";
+import React, { CSSProperties } from "react";
+
+import { SyncStatusEnum, SyncType } from "../../pages/types";
+
+type SyncCardProps = {
+  sync: SyncType;
+  removeSync: (sync: SyncType) => void;
+  editSync: (sync: SyncType) => void;
+  style?: CSSProperties;
+};
+
+const { Meta } = Card;
+
+const DetailsCutoff = 80;
+
+const SyncCard = ({
```

Comment:
similar to `NewMateCard.tsx` but needed to display different set of information.

Reply:
Probably. We could definitely refactor bunch of these into one, but let's cross that bridge when we get there since we don't know all the required info.